clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -2064,7 +2082,20 @@ struct CounterCoverageMappingBuilder
     createDecisionRegion(E, DecisionParams);
   }
 
+  /// Check if E belongs to system headers.
+  bool isExprInSystemHeader(const BinaryOperator *E) const {
```

Comment:
assert E is not nullptr?

Reply:
I assume each visitor (`visitBinLAnd` `visitBinLOr`) will not receive nullptr.